xnmt/xnmt_train.py

```diff
@@ -112,11 +112,12 @@ def pack_batches(self):
 
   def dynet_trainer_for_args(self, args):
     if args.trainer.lower() == "sgd":
-      trainer = dy.SimpleSGDTrainer(model_globals.dynet_param_collection.param_col, learning_rate = args.learning_rate)
+      trainer = dy.SimpleSGDTrainer(model_globals.dynet_param_collection.param_col)
```

Comment:
It's still fine to pass the learning rate through the constructor. Was this not working?